src/AppBundle/Service/EmailService.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: aurelie
+ * Date: 05/12/17
+ * Time: 09:49
+ */
+
+namespace AppBundle\Service;
+
+
+class EmailService
+{
+    protected $mailer;
+    protected $template;
+
+    public function __construct(\Swift_Mailer $mailer, \Twig_Environment $template)
+    {
+        $this->mailer = $mailer;
+        $this->template = $template;
+    }
+
+    public function sendMessageContact($contact)
+    {
+        $message = \Swift_Message::newInstance()
+            ->setSubject('Merci d\'avoir contacté HappySens')
+            ->setFrom($contact->getEmail(), $contact->getFirstName() . ' ' . $contact->getLastName())
+            ->setTo('famar.wcslyon@gmail.com')
```

Comment:
il est préférable d'utiliser un paramètre dans le config (ou le parameter.yml) pour définir le to du setTo

Reply:
modif faite